kongswap_adaptor/src/deposit/test/deposit_add_liquidity.rs

```diff
@@ -124,7 +124,7 @@ async fn run_add_liquidity_test(
                     subaccount: None,
```

Comment:
The fact that we call this constant `SELF_CANISTER_ID` makes it more difficult to review the PR. Let's maybe call it `TREASURY_MANAGER_CANISTER_ID`, so that it doesn't require broader context to find out what "self" is referring to. Feel free to do this later / independently of this PR.